### ISSUE TITLE
cmd+9 on mac is no longer valid

### ISSUE THREAD

**Issue by xc1427** [k--kato/vscode-intellij-idea-keybindings#82]
```
           {
                "key": "alt+9",
                "mac": "cmd+9",
                "command": "workbench.view.git",
                "when": "editorFocus",
                "intellij": "Open corresponding tool window (Git)"
            },
            {
                "key": "alt+numpad9",
                "mac": "cmd+numpad9",
                "command": "workbench.view.git",
                "when": "editorFocus",
                "intellij": "Open corresponding tool window (Git)"
            },
```

`workbench.view.git` should be `workben.view.scm`

**Comment by k--kato** [comment]
@xc1427 Thank you for your reporting.
Released v0.2.13, please check it.

**Comment by xc1427** [comment]
There are two instances of `workbench.view.git`, both of them should be changed to `.scm`, for the sake of OCD...
